_data-prepper/monitoring.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Monitoring
+nav_order: 33
+---
+
+# Monitoring Data Prepper with metrics
+
+You can monitor Data Prepper with metrics using [Micrometer](https://micrometer.io/). There are two types of metrics: JVM/system metrics and plugin metrics. [Prometheus](https://prometheus.io/) is used as the default metrics backend.
+
+## JVM and system metrics
+
+JVM and system metrics are runtime metrics that are used to monitor Data Prepper instances. They include metrics for classloaders, memory, garbage collection, threads, and others. For more information, see [JVM and system metrics](https://micrometer.io/docs/ref/jvm). 
+
+### Naming
+
+JVM and system metrics follow predefined names in [Micrometer](https://micrometer.io/docs/concepts#_naming_meters), such as `jvm_classes_loaded` and `jvm_memory_used`. 
+
+### Serving
+
+By default, metrics are served from the **/metrics/sys** endpoint on the Data Prepper server. The format is a text Prometheus scrape. The Data Prepper server port has a default value of `4900` that can can modify, and this port can be used for any frontend that accepts Prometheus metrics, such as [Grafana](https://prometheus.io/docs/visualization/grafana/). You can update the configuration to serve metrics to other registries like Amazon CloudWatch, that does not require or host the endpoint but publishes the metrics directly to CloudWatch.
+
+## Plugin metrics
+
+Each plugin (and, if necessary, each class within a plugin) is responsible for posting relevant metrics. Besides custom metrics introduced by specific plugins, the [Data Prepper API](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-api) introduces the following common metrics for plugin types: 
```

Comment:
I think the section is probably only developer oriented and should probably be removed. Or replace this with a small introduction.

Perhaps:

> Plugins report their own metrics. Data Prepper uses a naming convention to help with consistency in the metrics. Please note that plugin metrics do not use dimensions.

Please do keep the "Naming" and "Service" sub-sections.